### PR TITLE
Data factory and database changes for VH Reporting SDS project

### PR DESCRIPTION
**Pull request by fergduggan**
### Change description ###

Remove dependencies on Application Insights 
Add Device information from vhvideo Heartbeat table
Update business views in vhreporting
New device table in vhreporting

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: credential/vh-adf-mi-dev.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+	"name": "vh-adf-mi-dev",
+	"properties": {
+		"type": "ManagedIdentity",
+		"typeProperties": {
+			"resourceId": "/subscriptions/705b2731-0e0b-4df7-8630-95f157f0a347/resourcegroups/vh-reporting-infra-dev/providers/Microsoft.ManagedIdentity/userAssignedIdentities/vh-adf-mi-dev"
+		}
+	}
+}
```
Comment (chris-pateman) [inline credential/vh-adf-mi-dev.json:9]: This is not needed in here as it is added via the pipeline

File: credential/credential1.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+	"name": "credential1",
+	"properties": {
+		"type": "ManagedIdentity",
+		"typeProperties": {
+			"resourceId": "/subscriptions/705b2731-0e0b-4df7-8630-95f157f0a347/resourcegroups/vh-reporting-infra-dev/providers/Microsoft.ManagedIdentity/userAssignedIdentities/vh-adf-mi-dev"
+		}
+	}
+}
```
Comment (chris-pateman) [inline credential/credential1.json:9]: This is not needed in here as it is added via the pipeline

File: dataset/AsqlSource.json [hunk 3]
```diff
@@ -1,4 +1,6 @@
 {
+	"name": "AsqlSource",
+	"properties": {
```
Comment (chris-pateman) [inline dataset/AsqlSource.json:3]: You need to remove these as they are not needed

File: dataset/AsqlTarget.json [hunk 4]
```diff
@@ -1,4 +1,6 @@
 {
+	"name": "AsqlTarget",
+	"properties": {
```
Comment (chris-pateman) [inline dataset/AsqlTarget.json:3]: You need to remove these as they are not needed

File: factory/vh-datafactory-dev.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+	"identity": {
+		"type": "UserAssigned",
+		"userAssignedIdentities": {
+			"/subscriptions/705b2731-0e0b-4df7-8630-95f157f0a347/resourcegroups/vh-reporting-infra-dev/providers/Microsoft.ManagedIdentity/userAssignedIdentities/vh-adf-mi-dev": {
+				"clientId": "c0b23e22-aa84-40b0-bcb6-d67af410b70f",
+				"principalId": "575abcc6-53d2-4bcd-b26a-7bff89991e12"
+			}
+		}
+	}
+}
```
Comment (chris-pateman) [inline factory/vh-datafactory-dev.json:13]: This is not needed as it is deployed

File: linkedService/vhreporting-sds-kv-link.json [hunk 6]
```diff
@@ -1,10 +1,15 @@
 {
 	"name": "vhreporting-sds-kv-link",
+	"type": "Microsoft.DataFactory/factories/linkedservices",
```
Comment (chris-pateman) [inline linkedService/vhreporting-sds-kv-link.json:3]: You need to remove these as they are not needed

File: linkedService/vhreporting_blob.json [hunk 7]
```diff
@@ -2,11 +2,20 @@
...
+			"servicePrincipalKey": {
+				"type": "SecureString",
+				"value": "**********"
+			},
+			"tenant": "",
+			"encryptedCredential": "ew0KICAiVmVyc2lvbiI6ICIyMDE3LTExLTMwIiwNCiAgIlByb3RlY3Rpb25Nb2RlIjogIktleSIsDQogICJTZWNyZXRDb250ZW50VHlwZSI6ICJQbGFpbnRleHQiLA0KICAiQ3JlZGVudGlhbElkIjogIkRBVEFGQUNUT1JZQDAxRjVBQkVBLTMxNzktNEZGMy1BNEYxLUE3NjM0MDkwMjA5Nl8wMzA5OGRkNC0zODQ5LTQzOTEtODJkNi1hYWZjYmRmNWUwZmYiDQp9"
+		},
+		"annotations": []
 	}
 }
```
Comment (chris-pateman) [inline linkedService/vhreporting_blob.json:21]: This is not needed as it is deployed

File: managedVirtualNetwork/default/managedPrivateEndpoint/AzureBlobStorage958.json [hunk 8]
```diff
@@ -0,0 +1,10 @@
+{
+	"name": "AzureBlobStorage958",
+	"properties": {
+		"privateLinkResourceId": "/subscriptions/705b2731-0e0b-4df7-8630-95f157f0a347/resourceGroups/vh-core-infra-dev/providers/Microsoft.Storage/storageAccounts/vhcoreinfradev",
+		"groupId": "blob",
+		"fqdns": [
+			"vhcoreinfradev.blob.core.windows.net"
+		]
+	}
+}
```
Comment (chris-pateman) [inline managedVirtualNetwork/default/managedPrivateEndpoint/AzureBlobStorage958.json:10]: This is not needed as it is deployed

File: managedVirtualNetwork/default/managedPrivateEndpoint/vhadfendpoint.json [hunk 9]
```diff
@@ -0,0 +1,10 @@
+{
+	"name": "vhadfendpoint",
+	"properties": {
+		"privateLinkResourceId": "/subscriptions/705b2731-0e0b-4df7-8630-95f157f0a347/resourceGroups/vh-core-infra-dev/providers/Microsoft.Sql/servers/vh-core-infra-dev",
+		"groupId": "sqlServer",
+		"fqdns": [
+			"vh-core-infra-dev.database.windows.net"
+		]
+	}
+}
```
Comment (chris-pateman) [inline managedVirtualNetwork/default/managedPrivateEndpoint/vhadfendpoint.json:10]: This is not needed as it is deployed

File: managedVirtualNetwork/default/managedPrivateEndpoint/vhadfsdsendpoint.json [hunk 10]
```diff
@@ -0,0 +1,10 @@
+{
+	"name": "vhadfsdsendpoint",
+	"properties": {
+		"privateLinkResourceId": "/subscriptions/867a878b-cb68-4de5-9741-361ac9e178b6/resourceGroups/vh-infra-core-dev/providers/Microsoft.Sql/servers/vh-infra-core-dev",
+		"groupId": "sqlServer",
+		"fqdns": [
+			"vh-infra-core-dev.database.windows.net"
+		]
+	}
+}
```
Comment (chris-pateman) [inline managedVirtualNetwork/default/managedPrivateEndpoint/vhadfsdsendpoint.json:10]: This is not needed as it is deployed

File: pipeline/Child_AsqlToAsql.json [hunk 11]
```diff
@@ -1,4 +1,6 @@
 {
+	"name": "Child_AsqlToAsql",
+	"properties": {
```
Comment (chris-pateman) [inline pipeline/Child_AsqlToAsql.json:3]: remove as will break pipelines

File: pipeline/Master_PurgeData.json [hunk 12]
```diff
@@ -1,4 +1,6 @@
 {
+	"name": "Master_PurgeData",
+	"properties": {
```
Comment (chris-pateman) [inline pipeline/Master_PurgeData.json:3]: remove as will break pipelines

File: pipeline/Master_Staging.json [hunk 13]
```diff
@@ -1,4 +1,6 @@
 {
+	"name": "Master_Staging",
+	"properties": {
```
Comment (chris-pateman) [inline pipeline/Master_Staging.json:3]: remove as will break pipelines

File: publish_config.json [hunk 14]
```diff
@@ -0,0 +1 @@
+{"publishBranch":"adf_publish"}
```
Comment (chris-pateman) [inline publish_config.json:1]: not requried

File: trigger/Daily_Purge_Data.json [hunk 15]
```diff
@@ -1,28 +1,31 @@
 {
-	"annotations": [],
-	"runtimeState": "Started",
-	"pipelines": [
-		{
-			"pipelineReference": {
-				"referenceName": "Master_PurgeData",
-				"type": "PipelineReference"
+	"name": "Daily_Purge_Data",
+	"properties": {
```
Comment (chris-pateman) [inline trigger/Daily_Purge_Data.json:3]: remove as will break pipelines

File: trigger/Daily_VH_Video_Data_Load.json [hunk 16]
```diff
@@ -1,25 +1,28 @@
 {
-	"annotations": [],
-	"runtimeState": "Started",
-	"pipelines": [
-		{
-			"pipelineReference": {
-				"referenceName": "Master_Staging",
-				"type": "PipelineReference"
+	"name": "Daily_VH_Video_Data_Load",
+	"properties": {
```
Comment (chris-pateman) [inline trigger/Daily_VH_Video_Data_Load.json:3]: remove as will break pipelines

**Review by chris-pateman — CHANGES_REQUESTED** [review]

**Review by fergduggan — COMMENTED** [review]
Fixes look correct

**Comment by fergduggan** [comment]
Reopen with fixes

**Pull request closed without merging**